mempool/clist_mempool.go

```diff
@@ -418,17 +426,16 @@ func (mem *CListMempool) resCbFirstTime(
 			memTx.addSender(txInfo.SenderID)
 			mem.addTx(memTx)
 			mem.logger.Debug(
-				"added good transaction",
+				"added valid transaction to mempool",
```

Comment:
Any real reason for changing this log message?

Reply:
Same as above. Added "to mempool" to distinguish it from adding to cache.

Reply:
Reverted.